cmd/kubeadm/app/phases/certs/pkiutil/pki_helpers.go

```diff
@@ -123,23 +180,22 @@ func TryLoadCertAndKeyFromDisk(pkiPath, name string) (*x509.Certificate, *rsa.Pr
 	case *ecdsa.PrivateKey:
```

Comment:
Just curious, it doesn't matter if it is RSA or not anymore? Or we know that it is?